app/grandchallenge/products/views.py

```diff
@@ -241,10 +260,28 @@ def get_success_url(self):
         return reverse("products:product-list")
 
 
-class ProjectAirForm(PermissionRequiredMixin, CreateView):
-    template_name = "products/projectairfiles_form.html"
-    form_class = ProjectAirFilesForm
-    permission_required = f"{ProjectAirFiles._meta.app_label}.add_{ProjectAirFiles._meta.model_name}"
+class ProductsPostCreate(PostCreate):
```

Comment:
As is, these 3 new views need tests for permissions at the very least as you can edit any post with these views, and we need to ensure that a change made in the future here will not open up editing posts to users that do not have the correct permissions.

Reply:
I have added tests similar to the blogs_tests. I haven't really figured out completely yet how tests work, please have a look if it looks all right to you. And probably the CI checks will provide me a hint soon too.